# @brief   frontend YAML
# @date    2024-04-09 update
# @version 0.1.1

#topic subscriber------------------------------------------------
topic_sub: #
  cloud_sub_topic: /synced_no_ground_cloud #sensor_msgs/PointCloud2

#lidar odom setting-------------------------------------------------
lidar_odom:
  keyframe_distance: 2.0
  keyframe_num: 5
  single_scan_leaf_size: 1.0 #filter size
  local_map_leaf_size: 1.0
  registration_resolution: 1.0
